src/zc/form/browser/combinationwidget.rst

```diff
@@ -207,3 +207,107 @@ There's also a display version of the widget::
           </tr>
         </table>
 
+In case of a wrong amount of parameters, the missing_value is used::
+
+    >>> widget = CombinationDisplayWidget(IDemo['acceptable_count'], request)
+    >>> widget.setPrefix('field')
+    >>> widget.setRenderedValue(('10', '2', '3'))
+    >>> print widget() # doctest: +ELLIPSIS +NORMALIZE_WHITESPACE +REPORT_NDIFF
+    <input type='hidden' name='field.acceptable_count-marker' value='x' />
+        <table class="combinationFieldWidget">
+          <tr>
+                  <td class="label">
+                    <label for="field.acceptable_count.combination_00">
+                      <span>Minimum</span>
+                    </label>
+                  </td>
+              <td class="field">
+                <div class="widget">
+                </div>
+              </td>
+          </tr>
+          <tr>
+                  <td class="label">
+                    <label for="field.acceptable_count.combination_01">
+                      <span>Maximum</span>
+                    </label>
+                  </td>
+              <td class="field">
+                <div class="widget">
+                </div>
+              </td>
+          </tr>
+        </table>
+
+In case the parameter is not a sequence, the missing_value is used::
+
+    >>> widget = CombinationDisplayWidget(IDemo['acceptable_count'], request)
+    >>> widget.setPrefix('field')
+    >>> widget.setRenderedValue(10)
+    >>> print widget() # doctest: +ELLIPSIS +NORMALIZE_WHITESPACE +REPORT_NDIFF
+    <input type='hidden' name='field.acceptable_count-marker' value='x' />
+        <table class="combinationFieldWidget">
+          <tr>
+                  <td class="label">
+                    <label for="field.acceptable_count.combination_00">
+                      <span>Minimum</span>
+                    </label>
+                  </td>
+              <td class="field">
+                <div class="widget">
+                </div>
+              </td>
+          </tr>
+          <tr>
+                  <td class="label">
+                    <label for="field.acceptable_count.combination_01">
+                      <span>Maximum</span>
+                    </label>
+                  </td>
+              <td class="field">
+                <div class="widget">
+                </div>
+              </td>
+          </tr>
+        </table>
```

Comment:
TBH I'm not following any of these tests.

(What is the value of `missing_value`?  Where is it used?)

(Perhaps I'd understand better if I read the previous doctests.)

Reply:
I read the previous doctests but do not understand what the value of `missing_value` is and how it manifests in the result of the test.

`missing_value` is not set explicitly in the doctest. I believe, defining it in the interface with a remarkable value would make it easier to understand the test.

Reply:
This was a rather tricky one, as the missing value could be set, but did not alter anything in the output, as it was not passed on to the fields combined in `Combination`. 

Thanks for pointing that out, so this bug could be fixed. Have a look at https://github.com/zopefoundation/zc.form/pull/4/commits/06603c104c7e12d7c2e20ef9336ae41c7e85d110 if you want.